### ISSUE TITLE
Enable hospitals to update a list of inventory items

### ISSUE THREAD

**Issue by bl-nero** [Lifespark-Technologies/Infomed#20]
UI mockups: https://www.figma.com/file/amyQtf3gEnSBSmpXN6gQ67/CoCoMed?node-id=380%3A31. See the "Hospital admin logged in page" section.

**Comment by naijauser** [comment]
Help me out with some context here, this is going to be  a new page right? What kind of info goes into the update? Is the page to display the inventory already built? Are we sticking to this folder structure for the project, in terms to grouping components appropriately (is this page just going to be added to the components folder)?

**Comment by Lifespark-Technologies** [comment]
@bl-nero

**Comment by bl-nero** [comment]
@naijauser I updated the description with a link to UI mockups that should answer most of your questions. As for the folder structure — I intended to keep it like this for now, but if you have any suggestions, I'm open. Be aware, though, that https://github.com/Lifespark-Technologies/Infomed/pull/29 will put the entire frontend inside a larger folder structure that will contain a backend Django server.
